donjo-app/models/migrations/Migrasi_1909_ke_1910.php

```diff
@@ -125,5 +125,17 @@ public function up() {
 	  $this->db->query($sql);
 	  $sql = "ALTER TABLE `program_peserta` CHANGE `kartu_nik` `kartu_nik` VARCHAR(30) NOT NULL";
 	  $this->db->query($sql);
+	  
+	  // ubah/perbaiki struktur database, table artikel
+	  $this->db->query('ALTER TABLE artikel MODIFY isi TEXT NOT NULL Default NULL;');
+	  $this->db->query('ALTER TABLE artikel MODIFY gambar VARCHAR(200) NOT NULL Default NULL;');
```

Comment:
Sql seperti ini salah, kalau NOT NULL tidak boleh diberi default NULL.
Seharusnya berbentu sbb:
`$this->db->query('ALTER TABLE artikel MODIFY gambar VARCHAR(200) DEFAULT NULL;');`